doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,106 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes in detail the detection of foreign keys drops for MySQL databases, 
+using Atlas's linting capabilities.
+
+A foreign key constraint can be dropped from a table using the following command:
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_id;
+```
+
+Even though this action is not causing data loss and there might be good reasons to perform it on a production database, it can have negative and sometimes unpredictable side effects (some of them are explored below) which should be taken into consideration.
+Atlas's linter can be used to raise a flag when a foreign key drop is about to be executed. Simply run:
+```bash
+atlas migrate lint --dir file://migrations --dev-url "mysql://root:pass@localhost:3306/dev" --latest 1
+```
+The `--latest` flag tells atlas to inspect only the last migration file. If it contains a foreign key drop, the output will be something like:
+
+```bash
+20230305125128.sql: constraint deletion detected:
+
+	L1: Dropping foreign-key constraint "owner_id"
+```
+
+[Click here](/versioned/lint) for more information about how to use the Atlas CLI Linter.
```

Comment:
```suggestion
[Click here](/versioned/lint) for more information about how to use the Atlas Linter.
```

Reply:
Some of my comments were resolved, but the text was not changed (like this one).